taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = ("username", "license_number", "first_name", "last_name",)

```

Comment:
The `license_number` field is included in the `fields` tuple, but it may not exist in the default user model returned by `get_user_model()`. Ensure that the user model has been extended to include this field.